### DISCUSSION TITLE
什么值得买7合一 模板问题反馈评论区

### DISCUSSION THREAD

**Discussion by wjf0214** [wjf0214/qd-templates#16] [模板讨论]
什么值得买7合一 模板问题反馈评论区。
有问题可以下面反馈。

**Comment by benlong567** [comment]
![QQ图片20221231144301](https://user-images.githubusercontent.com/98813249/210127966-5800e296-a446-409b-8fba-245596ea6a5a.png)
大佬，日志里活动结果后面没有显示，是我设置有撒问题吗，还是模板本身没有

**Reply by wjf0214** [reply]
活动结果这个，有活动的时候才有。大部分时候没有。正常的

**Reply by benlong567** [reply]
好的，谢谢大佬回复

**Comment by stkroot** [comment]
脚本失效了
